src/server/plugins/engine/components/DatePartsField.test.ts

```diff
@@ -113,6 +118,178 @@ describe('Date parts field', () => {
 
     expect(conditonEvaluationStateValue).toBe('2024-12-31')
   })
+
+  describe('Form validation', () => {
```

Comment:
Should probably match the other tests

```suggestion
  describe('Validation', () => {
```